app/controller_services/shopping_lists_controller/destroy_service.rb

```diff
@@ -44,7 +44,7 @@ def destroy_and_update_aggregate_list_items
         # If shopping_list is the user's last regular shopping list, this will also
         # destroy their aggregate list
         shopping_list.destroy!
-        
```

Comment:
Well it's annoying there's a whitespace change but at least it fixes trailing whitespace.